must-call-checker/src/main/java/org/checkerframework/checker/mustcall/Stream.astub

```diff
@@ -132,6 +113,34 @@ class BufferedInputStream extends FilterInputStream {
     void close() throws IOException;
 }
 
+class FileOutputStream extends OutputStream {
+    FileOutputStream(String arg0) throws FileNotFoundException;
```

Comment:
In general in the future, can we avoid adding an entry for every method in a class? It makes it difficult to find what's actually annotated.